scripts/splitClasses.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs')
+
+const classes = JSON.parse(
+  fs.readFileSync('../src/assets/classes.json').toString()
+)
+
+/**
+ * Splits the category labels between a given number of instances.
+ * 
+ * If there more instances than categories, categories are cyclically reused.
+ */
+const main = () => {
+  const nInstances = process.argv[3] || 1
+  const current = process.argv[2] || nInstances
```

Comment:
I don't understand what `current` means here? And it does not seemed really used since populate call it with `1` all of time?